be/src/formats/orc/orc_chunk_reader.cpp

```diff
@@ -408,6 +408,9 @@ static void _try_implicit_cast(TypeDescriptor* from, const TypeDescriptor& to) {
         } else {
             from->type = LogicalType::TYPE_DECIMAL32;
         }
+    } else if (_broker_load_mode && !_strict_mode && t1 == LogicalType::TYPE_VARCHAR &&
```

Comment:
1. The issue number is wrong.
2. It's better to add a unit test case to this bug.